tests/test_menu_music.py

```
@@ -0,0 +1,263 @@
+    ev.type = menu_music_mod.pygame.MOUSEBUTTONDOWN
+    ev.button = button
+    ev.pos = pos
+    return ev
+
+
+class TestNavigation:
+    def test_right_arrow_increments_index(self):
+        menu = _make_menu()
+        menu.right_rect = Mock()
+        menu.right_rect.collidepoint.return_value = True
+        menu.back_rect  = Mock(); menu.back_rect.collidepoint.return_value = False
```

Ruff (0.15.9)

[error] Multiple statements on one line (semicolon)

(E702)